packages/rest/src/SimpleResource.ts

```diff
@@ -157,11 +162,15 @@ export default abstract class SimpleResource extends Entity {
   }
 
   /** Base endpoint but for sideEffects */
-  protected static endpointMutate() {
+  protected static endpointMutate(): RestEndpoint<
+    (this: RestEndpoint, params: any, body?: any) => Promise<any>,
```

Comment:
same here - why not a generic that defaults to unknown?
```suggestion
    (this: RestEndpoint, params: T, body?: any) => Promise<T>,
```
(I’m not sure where to put the `<T = unknown>` here)

Reply:
Another note: We are not changing types here but defining them explicitly.

Reply:
`unknown` definitely does match everything - `any` is just `unknown` with typechecking disabled.